cinema/views.py

```diff
@@ -12,26 +25,40 @@
     MovieDetailSerializer,
     MovieSessionDetailSerializer,
     MovieListSerializer,
+    OrderSerializer,
+    TicketSerializer
 )
 
 
 class GenreViewSet(viewsets.ModelViewSet):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
 
+    def list(self, request, *args, **kwargs):
+        queryset = self.get_queryset()
+        serializer = self.get_serializer(queryset, many=True)
+        return Response(serializer.data)
```

Comment:
The `list` method in `GenreViewSet` is redundant because it does not add any custom behavior to the default `list` method provided by `ModelViewSet`. Consider removing it unless you plan to add custom logic.